setup.py

```diff
@@ -111,12 +131,12 @@ def finalize_options(self):
         "Programming Language :: Cython",
         "Programming Language :: Python :: 3",
         "Topic :: Multimedia :: Graphics :: 3D Rendering",
-        "Topic :: Scientific/Engineering :: Physics"
+        "Topic :: Scientific/Engineering :: Physics",
     ],
-    install_requires=['numpy', 'matplotlib'],
+    install_requires=["numpy", "matplotlib"],
```

Comment:
In pyproject.toml we use `oldest-supported-numpy`. The project is thus built against numpy 1.x for all versions of Python where that is supported, and means in turn that `numpy<2` is required at runtime.

Reply:
Yes, we should keep `numpy<2.0` in runtime.
However, when I specify `numpy > 2.0` in build time, raysect's unit tests seem to be succeeded according to the [`conda` building process](https://github.com/conda-forge/staged-recipes/pull/27726).
Needless to say, the unit test does not seem to cover all raysect's functionalities.

Reply:
Also, some demo scripts were successfully executed even in `numpy==2.1.2`.